site/content/docs/user/loadbalancer.md

```diff
@@ -29,13 +29,13 @@ description: |-
 ### Create the metallb namespace
 
 {{< codeFromInline lang="bash" >}}
-kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/master/manifests/namespace.yaml
+kubectl apply -f <https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/namespace.yaml>
```

Comment:
this is a snippet of code, the url should not be wrapped

see preview https://deploy-preview-2687--k8s-kind.netlify.app/docs/user/loadbalancer/

Reply:
Ah dang, I am not even sure where it came from.

Reply:
Fixed